src/main/scala/no/ndla/articleapi/controller/ArticleController.scala

```diff
@@ -103,13 +103,20 @@ trait ArticleController {
         summary "Retrieves a list of all previously used tags in articles"
         notes "Retrieves a list of all previously used tags in articles"
         parameters(
+        queryParam[Option[Int]]("size").description("Limit the number of results to this many elements"),
```

Comment:
Jeg cacher en sortert liste over alle tags som er brukt i artikler. Denne listen er sortert fra mest brukt til minst brukt (`mostFrequentOccorencesDec` i `ReadService`).
Det vil si at man kan be om de 10 mest brukte taggene ved å returnere de 10 første innslagene i denne listen.

Alle requests for å hente tags, uavhengig av hvor mange som requestes, går altså mot samme cache